docs/LDES.md

```diff
@@ -0,0 +1,111 @@
+
+- the current Mandataris entity keeps its `mandaat:status`, but the `mandaat:einde` changes to `T`. Let's call this original Mandataris entity `O`.
+- a new Mandataris entity is created with the same content as the first, but without a `mandaat:einde` and with a `mandaat:start` set to `T`. This new Mandataris entity receives the new `mandaat:status` of 'verhinderd', lets call this new Mandataris entity `V`.
+- if one wants to model that another Mandataris replaces the first, another Mandataris entity `R` is created and the new 'verhinderd' entity `V` receives a `mandaat:isTijdelijkVervangenDoor` reference to `R`.
+
+This means that except for correcting mistakes and changing the `mandaat:einde`, Mandataris entities should never be modified.
+
+## Expected Peripheral Knowledge
+
+There is a substantial amount of data that is referenced in the entities on the LDES above but that is not expected to change often (or at all) and that isn't the responsibility of the LMB system to maintain. Let's call such data Peripheral Knowledge. Examples are decretale Bestuursorganen and Mandaten or Election results.
+
+Such data will NOT be published on the LMB LDES feeds, but can be referenced by instances on our LDES feed. Therefore we expect systems that use the LDES data to have this Peripheral Knowledge loaded themselves.
+
+This presents the question of how this peripheral knowledge reaches these different systems (and LMB itself). Because the data will not change often, a set of migrations will suffice for now. In the future, a central system could be created that provides its own LDES feed with this peripheral knowledge. Other systems can then use this feed to update their peripheral knowledge. This is something that can wait as we will likely only launch LMB centrally first (possibly one separately with heavy, heavy support). In this case, a migration or set of migrations will be good enough. Other options are:
+
+- an LDES as discussed earlier. We can use the caching headers here to clarify that the resource isn't expected to change for another month or so. We can also terminate an LDES when we know no more data is going to change, e.g. for election results.
+- we can make an interface (API, not LDES) where they just fetch the latest version.
+
+## LDES setup
+
+The LDES spec can be found [here](https://semiceu.github.io/LinkedDataEventStreams/).
+We will be using the implementation provided by [redpencil.io on their github](https://github.com/redpencilio/fragmentation-producer-service) to publish the LDES. It will be fed using our own service that monitors deltas on our SEAS instance.
+
+We'll use a time-fragmenter, one stream will be set up per type of instance that we will share, with pagination and the following setup for versioning:
+
+    @prefix dct: <http://purl.org/dc/terms/>.
```

Comment:
My comment before was not about adding the actual prefixes, it was just that the last example had a comment written about it and the first one not, but this is certainly clear as well.